blueman/main/Tray.py

```diff
@@ -0,0 +1,54 @@
+from importlib import import_module
+
+import logging
+
+from blueman.Functions import check_single_instance
+from blueman.main.AppletService import AppletService
+
+import gi
+gi.require_version('Gtk', '3.0')
+from gi.repository import GLib, Gio
+
+
+class BluemanTray(object):
+    def __init__(self):
+        check_single_instance("blueman-tray")
+
+        logging.basicConfig(level=logging.INFO)
+
+        applet = AppletService()
+
+        main_loop = GLib.MainLoop()
+
+        Gio.bus_watch_name(Gio.BusType.SESSION, 'org.blueman.Applet',
+                           Gio.BusNameWatcherFlags.NONE, None, main_loop.quit)
+
+        indicator_name = applet.GetStatusIconImplementation()
+        indicator_name = 'GtkStatusIcon'
```

Comment:
Ah, sorry, forgot to mention that. 🙈 

You can easily verify that it's the AppIndicator item by left-clicking on it. The GtkStatusIcon should open blueman-manager while AppIndicator would probably show the menu. Also, blueman-tray tells you which indicator it is using.